test/framework/kogitoapp.go

```diff
@@ -109,17 +116,22 @@ func crDeployExample(namespace string, kogitoAppDeployment KogitoAppDeployment)
 	return nil
 }
 
-func cliDeployExample(namespace string, kogitoAppDeployment KogitoAppDeployment) error {
-	cmd := []string{"deploy-service", kogitoAppDeployment.AppName, config.GetExamplesRepositoryURI()}
+func cliDeployService(namespace string, kogitoAppDeployment KogitoAppDeployment) error {
+	cmd := []string{"deploy-service", kogitoAppDeployment.AppName}
+
+	if gitSourceURI := kogitoAppDeployment.GitSourceURI; len(gitSourceURI) > 0 {
+		cmd = append(cmd, gitSourceURI)
+		cmd = append(cmd, "-c", kogitoAppDeployment.ContextDir)
```

Comment:
do we suppose there is always a `ContextDir` ? I am just thinking about that now ...

Reply:
Good point, will ad an empty check there too.